packages/v2ray.rb

```diff
@@ -2,43 +2,40 @@
-  version 'v4.28.2'
+  homepage 'www.v2fly.org'
+  version 'v5.3.0'
   license 'MIT'
   compatibility 'all'
 
-  case ARCH
-  when 'aarch64', 'armv7l'
-    source_url 'https://github.com/v2ray/v2ray-core/releases/download/v4.28.2/v2ray-linux-arm32-v7a.zip'
-    source_sha256 '9090cfc97418719e66467f296bfc2297776da14d3af127d23aa3bb196d7707e8'
-  when 'i686'
-    source_url 'https://github.com/v2ray/v2ray-core/releases/download/v4.28.2/v2ray-linux-32.zip'
-    source_sha256 '8b31f08f6e25609792f1d1ad6a4d9e35e4089a19ac85b30bd70b95bd48b8e9e3'
-  when 'x86_64'
-    source_url 'https://github.com/v2ray/v2ray-core/releases/download/v4.28.2/v2ray-linux-64.zip'
-    source_sha256 'e50e6818a743f842e1bd464f81ed3e42896f68f461718d9ba0f8d75107355c11'
-  end
+  source_url({
+    aarch64: 'https://github.com/v2fly/v2ray-core/releases/download/v5.3.0/v2ray-linux-arm32-v7a.zip',
+     armv7l: 'https://github.com/v2fly/v2ray-core/releases/download/v5.3.0/v2ray-linux-arm32-v7a.zip',
+       i686: 'https://github.com/v2fly/v2ray-core/releases/download/v5.3.0/v2ray-linux-32.zip',
+     x86_64: 'https://github.com/v2fly/v2ray-core/releases/download/v5.3.0/v2ray-linux-64.zip'
+  })
+
+  source_sha256({
+    aarch64: 'd782a7efa04dda771751c43bb7096c1fde0879bbd1df65e796895fa9baba5c74',
+     armv7l: 'd782a7efa04dda771751c43bb7096c1fde0879bbd1df65e796895fa9baba5c74',
+       i686: 'c3bc3da7e1a5288299230856bf3b3f63762794d47997dba50c23760c6e517712',
+     x86_64: '6cf089757128eac6d823e50e91e4dbe4f2492170d24b8721ee5dc6ec3f7d3c9c'
+  })
+
+  no_compile_needed
 
   def self.install
-    FileUtils.mkdir_p("#{CREW_DEST_PREFIX}/share/v2ray")
-    FileUtils.cp_r('.', "#{CREW_DEST_PREFIX}/share/v2ray")
-    FileUtils.chmod('u=x', "#{CREW_DEST_PREFIX}/share/v2ray/v2ray")
-    FileUtils.chmod('u=x', "#{CREW_DEST_PREFIX}/share/v2ray/v2ctl")
-    FileUtils.mkdir_p("#{CREW_DEST_PREFIX}/bin")
-    FileUtils.cd("#{CREW_DEST_PREFIX}/bin") do
-      FileUtils.ln_s("#{CREW_PREFIX}/share/v2ray/v2ray", 'v2ray')
-      FileUtils.ln_s("#{CREW_PREFIX}/share/v2ray/v2ctl", 'v2ctl')
-    end
+    FileUtils.mkdir_p %W[#{CREW_DEST_PREFIX}/bin #{CREW_DEST_PREFIX}/etc/v2ray #{CREW_DEST_PREFIX}/share/v2ray]
+    FileUtils.chmod 0o755, 'v2ray'
+    FileUtils.mv Dir['*'], "#{CREW_DEST_PREFIX}/share/v2ray"
+    FileUtils.ln_s "#{CREW_PREFIX}/share/v2ray/v2ray", "#{CREW_DEST_PREFIX}/bin/v2ray"
+    FileUtils.ln_s "#{CREW_PREFIX}/share/v2ray/config.json", "#{CREW_DEST_PREFIX}/etc/v2ray/config.json"
   end
 
   def self.postinstall
-    puts
-    puts 'To start using v2ray, type `v2ray`.'.lightblue
-    puts
-    puts 'You can use customer config. about how to use v2ray command, see https://www.v2ray.com/'.lightblue
-    puts 'If you want to remove v2ray'.lightblue
-    puts
-    puts 'crew remove v2ray'.lightblue
-    puts
+    puts <<~EOT.lightblue
+      To start using v2ray, type `v2ray`.
+
+      You can use customer config. about how to use v2ray command, see www.v2fly.org
```

Comment:
`To use a custom config with the v2ray command, see www.v2fly.org`

Reply:
See https://www.v2fly.org/en_US/v5/config/overview.html to learn about configuration?

Reply:
Or for more information, see https://www.v2fly.org/en_US/guide/start.html#novice-guide?

Reply:
Change made